tools/gen_internal_conf/src/main.rs

```diff
@@ -0,0 +1,355 @@
+extern crate clap;
```

Comment:
Thank you for your review. Parsing can't be done if you don't tell the parser what new content to parse. We don't know what the JSON would be like in the future. Maybe a whole section will be added or removed. To handle all situations, a lot of effort is needed. And still, it may not work when we really change the JSON. 
This commit already eliminated the effort to modify the options in the JSON file. We may be able to find a way to do it really automatically, but currently, the most important thing is to find a balance between the effort and the effect, right?